ambry-router/src/test/java/com.github.ambry.router/NonBlockingRouterTest.java

```diff
@@ -411,6 +413,7 @@ public void testRouterNoPartitionInLocalDC() throws Exception {
    */
   @Test
   public void testRequestResponseHandlerThreadExitFlow() throws Exception {
+    nettyByteBufLeakHelper.setDisabled(true);
```

Comment:
why was the leak helper disabled for this test case?

Reply:
There are tests in `NonBlockingRouterTest` that needs creates new ByteBuf by putting a blob to the in memory router but never release it. Those tests are going to break the leak detector helper. That's why I disable the leak detector helper for those tests.